Fix some command syntaxes and examples. (#4)
inspircd/inspircd-docs: docs/2/commands/_cmd_whois.md

<!-- This file contains a page fragment. Any changes will affect all pages that include it. -->

### `/WHOIS [<server>] <nick>[,<nick>]+`

Requests information about users who are currently connected with the specified nicks:

> If the `<server>` parameter is specified then remote information will be fetched about the user if they are not on the local server. Note, however, that if the `<server>` parameter is specified you can not specify multiple nicks, only one nick can then be specified.

Reviewer: ```suggestion
If the `<server>` parameter is specified then only one `<nick>` can be specified and remote information will be fetched about the user if they are not on the local server.
```

/WHOIS Adam
```

Requests remotely available information about Adam by specifying the server they're on:

```plaintext
/WHOIS irc.example.com Adam
```

> Requests remotely available information about Adam by repeating their nick as follows:

Reviewer: ```suggestion
Requests remotely available information about Adam by repeating their nick:
```
